### DISCUSSION TITLE
Training on charged systems.

### DISCUSSION THREAD

**Discussion by JSLJ23** [ACEsuit/mace#461] [Q&A] [answered]
I noticed that the MACE-OFF23 was originally trained on a subset of SPICE that only contained neutral molecules. Was there a reason for the exclusion of charged molecules and are there any modifications that are required to support training on both neutral and charged molecules?

**Comment by gabor1** [comment, marked as answer]
yes, the currently published MACE model has no explicit analytic coulomb interactions. You can of course train it on the entire spice dataset including charged systems, but it will be much less accurate, because the different energy of charged and uncharged systems is poorly reflected  in just the geometry only. We are working on adding explicit charges to MACE, at which point the entire database will be refitted. Not far away now (it works for us in-house already, and is undergoing testing before release)

**Reply by JSLJ23** [reply]
Thank you for the prompt reply.
Just curious at a high level and in brief, what changes must be made to the MACE model to support these charged systems?

**Reply by gabor1** [reply]
of course there are many ways in one can go about this. We are using a charge-self-consistent approach, in which the atom energy is a function of local geometry and field, and this is iterated to self-consistency.

**Reply by yumengzhang1998** [reply]
Hi, will this function be published any soon?

**Reply by gabor1** [reply]
months I would say.
